### ISSUE TITLE
FR: Propagate hyperparameters of optimizers in kedro

### ISSUE THREAD

**Issue by stroblme** [cirKITers/Split-Optimization#25]
**Is your feature request related to a problem? Please describe.**
Cannot set learning rate for optimizer(s) individually

**Describe the solution you'd like**
Introduce individual learning rate for optimizer(s) together with a concept for handling optimizer specific hyperparameters

**Describe alternatives you've considered**
N/A

**Additional context**
N/A
